bin/buildBlueprints.js

```diff
@@ -69,38 +69,39 @@ function loadBlueprintsFromPath(filePath, isProduction) {
 
 function loadDefaultConfigs(options) {
   console.log('...using default configs');
-  if (options.runTest) {
-    console.log('...Setting up tests:');
-    var config = _.merge(
-      {},
-      configs.DefaultTestingConfig,
-      { webpack: { entry: getWebpackEntryForTest('./') } }
-    );
-    return [ config ];
-
-  } else if (options.client) {
-    console.log('...client');
-    return [ configs.getClientConfig(options.production) ];
-
-  } else if (options.server) {
-    console.log('...server');
-    return [ configs.getServerConfig(options.production) ];
-
-  } else if (options.clientAndServer) {
-    console.log('...both');
-    return [
-      configs.getClientConfig(options.production),
-      configs.getServerConfig(options.production),
-    ];
+  var production = options.env === PRODUCTION_ENV;
+  switch (options.target) {
+    case TARGETS.TEST:
+      console.log('...Setting up tests:');
+      var config = _.merge(
+        {},
+        configs.DefaultTestingConfig,
+        { webpack: { entry: getWebpackEntryForTest('./') } }
+      );
+
+      return [ config ];
+    case TARGETS.CLIENT:
+      console.log('...client');
+      return [ configs.getClientConfig(production) ];
+    case TARGETS.SERVER:
+      console.log('...server');
+      return [ configs.getServerConfig(production) ];
+    default:
+      console.log('...both');
+      return [
+        configs.getClientConfig(production),
+        configs.getServerConfig(production),
+      ];
+
   }
 }
 
 function makeConfig(options) {
   var builds;
   var extensions = {};
 
-  if (options.blueprintsPath && !options.ignoreBlueprints && !options.runTest) {
-    var blueprints = loadBlueprintsFromPath(options.blueprintsPath, options.production);
+  if (options.blueprintsPath && !options.ignoreBlueprints) {
```

Comment:
Sounds good. In general this a bit of a breaking change versus the old api. For example 2X uses a custom blueprints that is not aware of tests, so there's a tiny bit of migration in your configs / `package.json` that needs to happen. Would you be up for adding some quick migration notes to the readme when the patch settles?